bundles/io/org.openhab.io.rest/src/main/java/org/openhab/io/rest/internal/listeners/ResourceStateChangeListener.java

```diff
@@ -54,6 +54,7 @@
 	
 	static ScheduledFuture<?> executorFuture;
 	
+	protected Item lastChange;
```

Comment:
Have you tested what happens if many things change at the same time?  Do you still get all the events?